dlt/extract/source.py

```diff
@@ -642,22 +642,17 @@ class DltSource(Iterable[TDataItem]):
     * You can use a `run` method to load the data with a default instance of dlt pipeline.
     * You can get source read only state for the currently active Pipeline instance
     """
-    def __init__(self, name: str, section: str, schema: Schema, resources: Sequence[DltResource] = None) -> None:
-        self.name = name
+    def __init__(self, section: str, schema: Schema, resources: Sequence[DltResource] = None) -> None:
```

Comment:
could we swap schema with section? it feels more natural - schema is the most important arg

Reply:
done